src/components/patient/PatientDetails.tsx

```diff
@@ -12,13 +13,27 @@ import ApiErrorMessage from '../error/ApiErrorMessage'
 import NotFound from '../error/NotFound'
 import PageHeader from '../shell/PageHeader'
 import AddressPanel from './AddressPanel'
+import ConfirmationModal from './ConfirmationModal'
 import DemographicsPanel from './DemographicsPanel'
 import IdentifiersPanel from './IdentifiersPanel'
 import RegisteringFacilityPanel from './RegisteringFacilityPanel'
 import RelationshipPanel from './RelationshipPanel'
 import SubHeading from './SubHeading'
 
+export interface UpdatedFields {
+  id: number
+  field: string
+  original: any
+  new: any
```

Comment:
```suggestion
  oldFieldValue: any
  newFieldValue: any
```
"new" is a reserved keywoed for object instanciation.